### PR TITLE
Update xenserver.rst

### PR DESCRIPTION
**Pull request by andrijapanicsb**
The focus of this PR is on updating the XenServer upgrade documentation, but also some other updates are done in sense of supported XS versions etc.

Tested by upgrading XS 7.1 to XCP-ng 7.6

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
the updated page:  https://acs-www.shapeblue.com/docs/WIP-PROOFING/pr140/installguide/hypervisor/xenserver.html#upgrading-xenserver-versions

**Comment by rhtyd** [comment]
cc @Spaceman1984

**Comment by nvazquez** [comment]
Hi @andrijapanicsb is this one ready for review?

**Comment by andrijapanicsb** [comment]
@nvazquez yes, should be fine.

File: source/installguide/hypervisor/xenserver.rst [hunk 1]
```diff
@@ -31,12 +31,17 @@ System Requirements for XenServer Hosts
...
+    -  XenServer 7.0 
+    -  XenServer 7.1
+    -  XenServer 7.5
+    -  XenServer 8.0
+    -  XenServer 8.1
+    -  XCP-ng 7.4
+    -  XCP-ng 7.5
+    -  XCP-ng 7.6
+    -  XCP-ng 8.0
+    -  XCP-ng 8.1
```
Comment (nvazquez) [inline source/installguide/hypervisor/xenserver.rst:43]: Missing 8.2 - also, should we add `.0` on each of them? As per @weizhouapache's comment on other PR: https://github.com/apache/cloudstack-documentation/pull/272#discussion_r863680841

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Pull request merged**